src/OrchardCore.Modules/OrchardCore.Twitter/Services/TwitterClient.cs

```diff
@@ -9,12 +9,11 @@ namespace OrchardCore.Twitter.Services
 {
     public class TwitterClient
     {
-        private readonly HttpClient _client;
+        internal static HttpClient _client = new HttpClient();
```

Comment:
Regression issue for https://github.com/OrchardCMS/OrchardCore/issues/6373 is here.

Because over here https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Modules/OrchardCore.Twitter/Startup.cs

it's registered with the `HttpClientFactory`

So either needs to go back to using the factory, or be registered correctly. Preference @sebastienros ?